config/sync/core.entity_form_display.node.article.default.yml

```diff
@@ -8,34 +8,53 @@ dependencies:
     - field.field.node.article.field_a_hero_image
     - field.field.node.article.field_a_summary
     - field.field.node.article.field_a_teaser_image
+    - field.field.node.article.field_ready_to_review
     - node.type.article
     - workflows.workflow.content_workflow
   module:
     - comment
     - content_moderation
+    - field_group
     - media_library
     - paragraphs
     - path
+third_party_settings:
+  field_group:
+    group_ready_to_review:
+      children:
+        - field_ready_to_review
+      parent_name: ''
+      weight: 13
+      format_type: details_sidebar
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+        description: ''
```

Comment:
Add some description text explaining what this field is for. Both changes will need to be done on article and recipe.

Reply:
Done